client/app/bundles/momentCard/components/MomentCardDraft.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from 'react';
+import Tag from '../../shared/components/Tag';
+import css from './MomentCard.scss';
+
+type MomentCardDraftState = {};
+
+type MomentCardDraftProp = {
+};
+
+export default class MomentCardDraft
+  extends React.Component<MomentCardDraftProp, MomentCardDraftState> {
+  props: MomentCardDraftProp;
+  state: MomentCardDraftState;
+
+  render() {
+    return (
+      <div className={css.draft}>
+        <Tag label={'DRAFT'} />
```

Comment:
Allow the ability to change the wording of "Draft" to support multiple languages.